### DISCUSSION TITLE
Opening a projects link in android beta results in infinite loop

### DISCUSSION THREAD

**Discussion by DLMousey** [community/community#102927] [Mobile] [answered]
### Select Topic Area

Bug

### Body

# What happens

When I click on a link to a GitHub projects board from inside the android app a browser window opens and an infinite redirect loop (presumably projects trying to redirect me back into the app, then the app trying to open projects) starts, requiring the app to be closed via multitasking

# What I expected to happen

Either a screen telling me projects aren't supported on android yet, or a web browser opening successfully on the project URL

# Device specifics
- Google Pixel 8
- Android version 14, build number UQ1A.240105.004
- GitHub beta app, version 1.148.0 (also occurs on version 1.147.0)

# Things I've tried
- Attempted on WiFi, 4g and 5g
- Cleared cache
- Cleared storage and signed in again
- Updated from beta 1.147.0 to 1.148.0

# Video

https://github.com/community/community/assets/8281699/b77d8440-d10e-4c7a-af57-a5fc22e1eb7b

(The link above opens fine in Firefox)
![Screenshot_20240131-205520](https://github.com/community/community/assets/8281699/3d8212a9-6f3c-451b-9013-a264186ae80b)

**Comment by alcere** [comment]
Hi @DLMousey, Thank you for reporting the problem. We apologize for the inconvenience. Could you please tell us which browser and version you are using so we can try to reproduce the issue? Thank you for your help.

**Reply by DLMousey** [reply]
I'm using Firefox 122.0.1 (Build #2016000543), haven't tested with Chrome as a default  unfortunately

**Reply by alcere** [reply, marked as answer]
Hi @DLMousey! We’ve shipped several updates to the app that may have addressed this issue. Have you noticed any improvement?

**Reply by DLMousey** [reply]
Can confirm this is now fixed @alcere 😃

**Comment by pamelaacme** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> # What happens
> 
> When I click on a link to a GitHub projects board from inside the android app a browser window opens and an infinite redirect loop (presumably projects trying to redirect me back into the app, then the app trying to open projects) starts, requiring the app to be closed via multitasking
> 
> # What I expected to happen
> 
> Either a screen telling me projects aren't supported on android yet, or a web browser opening successfully on the project URL
> 
> # Device specifics
> - Google Pixel 8
> - Android version 14, build number UQ1A.240105.004
> - GitHub beta app, version 1.148.0 (also occurs on version 1.147.0)
> 
> # Things I've tried
> - Attempted on WiFi, 4g and 5g
> - Cleared cache
> - Cleared storage and signed in again
> - Updated from beta 1.147.0 to 1.148.0
> 
> # Video
> 
> https://github.com/community/community/assets/8281699/b77d8440-d10e-4c7a-af57-a5fc22e1eb7b
> 
> (The link above opens fine in Firefox)
> ![Screenshot_20240131-205520](https://github.com/community/community/assets/8281699/3d8212a9-6f3c-451b-9013-a264186ae80b)
>

**Comment by kedirabdukerim202** [comment]
My name is kedir Abdukerim Ahmed 
My phone number +2510912160267
My email kedirabdukerim202@gmail.com 
My telegram account no work pls help me
